src/App.css

```diff
@@ -80,4 +87,4 @@
 
 .link-homepage:hover {
   color: #c30000;
-}
+}
```

Comment:
Just putting it here since it is a CSS comment:
The comment text color currently makes it so the comments can't be read, as its white on white